test/unit/spec/DeviceFingerprint_spec.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-global-assign */
```

Comment:
Nit: Can you scope this to the function that does global variable assignments, as opposed to the whole file?